computingservices/DedupeServices/services/s3documentservice.py

```diff
@@ -72,6 +73,12 @@ def gets3documenthashcode(producermessage):
         filepath = path.splitext(filepath)[0] + extension
     response = requests.get("{0}".format(filepath), auth=auth, stream=True)
     reader = None
+    # Get pagecount for replaced attachments of incompatible files
```

Comment:
The reasoning here is that if the s3 extension is a pdf, and the original document extension is not in file_conversion_types and is not a pdf, then this is a replacement pdf of an incompatible (non-redactable) file.